Tests/ML/histopathology/utils/test_metrics_utils.py

```diff
@@ -37,7 +47,17 @@ def assert_equal_lists(pred: List, expected: List) -> None:
                   [Tensor([[0.1, 0.0, 0.2, 0.15]]),
                   Tensor([[0.10, 0.18, 0.15, 0.13]]),
                   Tensor([[0.25, 0.23, 0.20, 0.21]]),
-                  Tensor([[0.33, 0.31, 0.37, 0.35]])]
+                  Tensor([[0.33, 0.31, 0.37, 0.35]])],
+             ResultsKey.TILE_X:
+                  [Tensor([200, 200, 424, 424]), 
+                  Tensor([200, 200, 424, 424]),
+                  Tensor([200, 200, 424, 424]), 
+                  Tensor([200, 200, 424, 424])],
+             ResultsKey.TILE_Y: 
+                  [Tensor([200, 424, 200, 424]),
```

Comment:
these coordinates look all the same apart this one. I couldn't follow the reasoning. If we're using these to test the maps wouldn't be better to assign different coordinates to each tile?

Reply:
The reasoning is as follows: I am only using the coordinates from the first slide for the test, and the other coordinates are just to map to the same sizes in the Dict.